src/foam/nanos/crunch/UserCapabilityJunctionRefine.js

```diff
@@ -250,6 +254,11 @@ foam.CLASS({
       name: 'gracePeriod',
       includeInDigest: true,
       section: 'ucjExpirySection'
+    },
+    {
+      class: 'Boolean',
+      name: 'skipEditBehaviour',
```

Comment:
Is this to override the permission check?

Reply:
This is to update UCJ data without invoking edit behaviour, and if you don't have permission but set this to true anyway you get an authorization exception. The purpose is say an approval request was created as a side-effect of edit behaviour, you wouldn't want updating the data and approving to generate another approval request.